index.html

```diff
@@ -3,6 +3,11 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta property="og:title" name="title" content="Lyrics Quizzer" />
+  <meta property="og:type" content="website" />
+  <meta property="og:image" name="image" content="images/lyrics-quizzer.png" />
```

Comment:
This will need to be a complete absolute url `https://amyaitran.github.io/lyrics-quizzer/images/lyrics-quizzer.png`